src/EditorFeatures/CSharpTest/InitializeParameter/AddParameterCheckTests.cs

```diff
@@ -404,19 +404,6 @@ partial class C
             }.RunAsync();
         }
 
-        [Fact, Trait(Traits.Feature, Traits.Features.CodeActionsInitializeParameter)]
-        public async Task TestNotOnExternMethod()
-        {
-            var code = @"
-using System;
-
-class C
-{
-    extern void M([||]string s);
-}";
-            await VerifyCS.VerifyRefactoringAsync(code, code);
-        }
-
```

Comment:
Exact duplicate of:

https://github.com/dotnet/roslyn/blob/ee340b015525af93ab8f453acf0167f2f170c04d/src/EditorFeatures/CSharpTest/InitializeParameter/AddParameterCheckTests.cs#L182-L193